seirmo/model.py

```diff
@@ -0,0 +1,37 @@
+#
+# ForwardModel Class
+#
+
+
+class ForwardModel:
+    """ForwardModel Class:
+
+    Abstract base class for the SEIR model class.
+
+    Parameters
+    ----------
+    value: numeric, optional
+        example of value
+
+    """
+
+    def __init__(self):
```

Comment:
Could you change the `__init__` to 
```
def __init__(self):
   super(ForwardModel, self).__init__()
```

This makes sure that the inherited class is initialised properly. (Again this is done by Python under the hood, but it's nicer to make this explicit.